haddock-library/test/Documentation/Haddock/ParserSpec.hs

```diff
@@ -99,9 +102,21 @@ spec = do
       it "doesn't parse empty identifiers" $ do
         "``" `shouldParseTo` "``"
 
-      it "can parse infix identifiers" $ do
+      it "can parse an identifier in infix notation enclosed within backticks" $ do
         "``infix``" `shouldParseTo` "`" <> DocIdentifier "infix" <> "`"
 
+      it "can parse identifiers in infix notation enclosed within single quotes" $ do
+        "'`infix`'" `shouldParseTo` "`" <> DocIdentifier "infix" <> "`"
```

Comment:
I think this is the most problematic of all of these test failures, as exactly this syntax is described http://haskell-haddock.readthedocs.io/en/latest/markup.html#hyperlinked-identifiers.

Here's the test output:

       expected: DocAppend (DocString "`") (DocAppend (DocIdentifier "infix") (DocString "`"))
        but got: DocAppend (DocString "'") (DocAppend (DocIdentifier "infix") (DocString "'"))

Reply:
I just saw that this has been reported in https://github.com/haskell/haddock/issues/780.